src/components/Input/index.tsx

```diff
@@ -0,0 +1,16 @@
+import InternalInput, { InputProps } from './Input';
+
+import TextField from './TextField/TextField';
+
+export { TextFieldProps } from './TextField/TextFieldProps';
+
+interface CompoundedComponent
+    extends React.ForwardRefExoticComponent<InputProps> {
```

Comment:
langsung destructure importnya
```suggestion
    extends ForwardRefExoticComponent<InputProps> {
```